tests/game/helsknight-rare.test.ts

```diff
@@ -0,0 +1,49 @@
+import {describe, expect, test} from '@jest/globals'
+import HelsknightRare from 'common/cards/alter-egos/hermits/helsknight-rare'
+import EthosLabCommon from 'common/cards/default/hermits/ethoslab-common'
+import TNT from 'common/cards/default/single-use/tnt'
+import {attack, endTurn, playCardFromHand, testGame} from './utils'
+
+describe('Test Hels Trap Hole', () => {
+	test('Test Trap Hole with TNT', () => {
+		testGame(
+			{
+				playerOneDeck: [EthosLabCommon, TNT],
+				playerTwoDeck: [HelsknightRare],
+				saga: function* (game) {
+					yield* playCardFromHand(game, EthosLabCommon, 'hermit', 0)
+
+					yield* endTurn(game)
+
+					yield* playCardFromHand(game, HelsknightRare, 'hermit', 0)
+
+					yield* attack(game, 'secondary')
+
+					yield* endTurn(game)
+
+					yield* playCardFromHand(game, TNT, 'single_use')
+
+					yield* attack(game, 'single-use')
+
+					// Check TNT attack log was sent by current player
+					expect(
+						game.chat.find(
+							(message) =>
+								message.message.TYPE === 'ListNode' &&
+								message.message.nodes.find(
+									(node) =>
+										node.TYPE === 'PlaintextNode' &&
+										node.text.includes('backlash damage'),
+								),
+						)?.sender.id,
+					).toBe(game.currentPlayerEntity)
```

Comment:
I don't think this is a good way to check if TNT was used. I think having a dependency on the logging system in the game tests will make this very confusing to fix if we ever refactor logs or the TNT log changes. I would prefer if you checked the HP of the hermits.

Reply:
This check was to verify the current player sent the attack log instead of the opponent player (the new owner of TNT), I can remove this test and system dependency now that the sender is fixed

Reply:
I see. I'm not sure I see this as a valuable test so I prefer if you removed it.

Reply:
I actually just meant the one part with the TNT log check - I added the rest of the test back because I think its really useful.